.github/workflows/kitchen-tests.yml

```diff
@@ -138,28 +116,42 @@ jobs:
           TF_VAR_site_certificate_file: /tmp/tls.cer
           TF_VAR_site_certificate_key_file: /tmp/tls.key
           USE_GKE_GCLOUD_AUTH_PLUGIN: true
-      - name: Clean kitchen and terraform files
-        run: |
-          rm -rf .kitchen
-          rm -rf examples/*/.terraform
-          rm -rf examples/*/terraform.tfstate*
-          rm -f ~/.kube/config
-          ls -la ~/.kube || :
-      - name: Post-check dirty dirs in self-hosted
-        run: |
-          ls -la .
-          ls -la examples/sample-jhub
-          ls -la examples/sample-jhub-nfs
-          ls -la ~
-
-  # Clean up dangling images in self-hosted runner to avoid filling up docker disk
-  docker-cleanup:
-    needs: [test]
-    runs-on: self-hosted
-    steps:
-      - name: Clean up docker images
-        run: |
-          docker images --filter=reference='brownccv/kitchen-terraform'
-          docker images --filter=reference='brownccv/kitchen-terraform' --filter "dangling=true" | grep -E "brownccv/kitchen-terraform" | awk '{ print $3 }' | xargs docker rmi || :
-          echo "After cleanup"
-          docker images --filter=reference='brownccv/kitchen-terraform'
+
+  # test-sample-jhub-nfs:
```

Comment:
I wonder if instead of having this commented out, we just had line 122 with something like
`if: false  #set to true to enable test`

Reply:
Sounds good! Maybe we can also set something like `if: "contains(github.event.commits[0].message, '[test nfs]')"`?